readme.md

```diff
@@ -211,11 +217,17 @@ Migration from version 1.x
 
 Changelog
 =========
+
+3.0.1-SNAPSHOT
```

Comment:
This will be version 4.0.0

The plugin uses [semantic versioning](http://semver.org/)

I can release usually within a day or so of merging.